libr/bin/format/pe/pe.c

```diff
@@ -235,14 +235,15 @@ static int PE_(r_bin_pe_parse_imports)(struct PE_(r_bin_pe_obj_t)* bin, struct r
 					break;
 				snprintf (import_name, PE_NAME_LENGTH, "%s_%s", dll_name, name);
 			}
-			if (!(*importp = realloc (*importp, (*nimp+1) * sizeof(struct r_bin_pe_import_t)))) {
+			if (!(*importp = realloc (*importp, (*nimp + 1) * sizeof(struct r_bin_pe_import_t)))) {
 				r_sys_perror ("realloc (import)");
 				return R_FALSE;
 			}
 			memcpy((*importp)[*nimp].name, import_name, PE_NAME_LENGTH);
 			(*importp)[*nimp].name[PE_NAME_LENGTH] = '\0';
 			(*importp)[*nimp].vaddr = FirstThunk + i * sizeof (PE_DWord);
-			(*importp)[*nimp].paddr = PE_(r_bin_pe_vaddr_to_paddr)(bin, FirstThunk) + i * sizeof(PE_DWord);
+			(*importp)[*nimp].vaddr += bin->nt_headers->optional_header.ImageBase;
```

Comment:
I'm not very familiar with this code, but I think in the future this needs a better fix (like a function pe_rva_to_vaddr) because only sometimes you have to add the ImageBase.